egs/wsj/s5/steps/nnet3/align.sh

```diff
@@ -124,7 +124,7 @@ ivector_opts=
 if [ ! -z "$online_ivector_dir" ]; then
   ivector_period=$(cat $online_ivector_dir/ivector_period) || exit 1;
   # note: subsample-feats, with negative n, will repeat each feature -n times.
-  ivector_opts="--online-ivectors=scp:$online_ivector_dir/ivector_online.scp --online-ivector_period=$ivector_period"
+  ivector_opts="--online-ivectors=scp:$online_ivector_dir/ivector_online.scp --online-ivector-period=$ivector_period"
```

Comment:
Modifying the way this option is specified does not change the alignments, so I guess the option parser in c++ is immune this mis-specification.